src/frontends/pytorch/src/op/grid_sampler.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/grid_sample.hpp"
```

Comment:
Please include opset10 instead

Reply:
@mvafin gives opposite recommendation to use includes for operation directly to speed up compilation time

Reply:
no, that is not right. For example, in model creation sample we include operations via opset include. We are users of this API. `openvino/op` is more like internal API that can be changed, etc.

Reply:
Moreover, opset include is more logical and convenient for us from users perspective.

Reply:
Lets leave individual operation includes for now

Reply:
Agreed to leave as is.